site-src/guides/implementers-guide.md

```diff
@@ -0,0 +1,367 @@
+# Gateway API Implementer's Guide
```

Comment:
Nit: This location doesn't feel quite right. So far all our guides are targeted at users of the API, and this one is decidedly different than those other ones. Maybe this belongs under the "Reference" tab instead?